### PR TITLE
[Issue-147] lvremove might failed on big scale

### PR DESCRIPTION
**Pull request by AnastasiaShemyakinskaya**
Add `RunCmdWithAttempts` function, so we can run `lvremove` with retries

## Purpose
Solves https://github.com/dell/csi-baremetal/issues/147

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [x] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [x] All comments are resolved
- [x] PR validation is passed
- [x] Custom CI is passed

## Testing
_Link to custom CI build_

### CODE REVIEW

File: pkg/base/command/cmdexec.go [hunk 1]
```diff
@@ -51,6 +52,31 @@ func (e *Executor) SetLevel(level logrus.Level) {
...
+// Returns stdout as string, stderr as string and golang error if something went wrong
+func (e *Executor) RunCmdWithAttempts(cmd interface{}, attempts int) (string, string, error) {
+	ll := e.log.WithFields(logrus.Fields{
+		"method": "RunCmdWithAttempts",
+	})
+	var (
+		stdout string
+		stderr string
+		err    error
+		ticker = time.NewTicker(500 * time.Millisecond)
```
Comment (rususlasan) [inline pkg/base/command/cmdexec.go:66]: Let's make timeout configurable too.

Comment (AnastasiaShemyakinskaya) [reply pkg/base/command/cmdexec.go]: Done

**Review by rususlasan — APPROVED** [review]

File: pkg/base/linuxutils/lvm/lvm_test.go [hunk 2]
```diff
@@ -156,15 +157,14 @@ func TestLinuxUtils_LVRemove(t *testing.T) {
...
 	err = l.LVRemove(fullLVName)
 	assert.Nil(t, err)
-
-	e.OnCommand(cmd).Return("", "Failed to find logical volume", expectedErr).Times(1)
+	e.OnCommandWithAttempts(cmd, 5, 500*time.Millisecond).Return("", "Failed to find logical volume", expectedErr).Times(1)
 	err = l.LVRemove(fullLVName)
 	assert.Nil(t, err)
 
-	e.OnCommand(cmd).Return("", "", expectedErr).Times(1)
+	e.OnCommandWithAttempts(cmd, 5, 500*time.Millisecond).Return("", "", expectedErr).Times(1)
```
[the remark this answers is not in the document]
Comment (AnastasiaShemyakinskaya) [reply pkg/base/linuxutils/lvm/lvm_test.go]: Fixed

File: pkg/base/command/cmdexec.go [hunk 3]
```diff
@@ -51,6 +52,31 @@ func (e *Executor) SetLevel(level logrus.Level) {
...
+// Returns stdout as string, stderr as string and golang error if something went wrong
+func (e *Executor) RunCmdWithAttempts(cmd interface{}, attempts int, timeout time.Duration) (string, string, error) {
+	ll := e.log.WithFields(logrus.Fields{
+		"method": "RunCmdWithAttempts",
+	})
+	var (
+		stdout string
+		stderr string
+		err    error
+		ticker = time.NewTicker(timeout)
```
[the remark this answers is not in the document]
Comment (AnastasiaShemyakinskaya) [reply pkg/base/command/cmdexec.go]: Ticker was removed

**Review by painhardcore — APPROVED** [review]

**Pull request merged**